css/main.css

```diff
@@ -22,6 +23,7 @@ html {
 body,
 main {
   background: transparent;
+  margin: 8px;
```

Comment:
adding a bit of extra spacing, mostly for aesthetic reasons. Mobile view looked especially tight.

Reply:
would be helpful to comment this in the code so that the next person modifying knows what they'll affect if they change it

same for other comments